README.md

```diff
@@ -235,3 +236,6 @@ As mentioned in the Uprising rules, a Lord may call "Uprising" at the start of t
 
 If a Lad does not call "Uprising" or does not perform the necessary number of flips, the Uprising nail is reset along with that Lad's nail (they must hammer their current starting nail into the stump if it is not already finished before beginning a new one).
 
+## X - Sparks
+
+If a spark is observed by any player when the hammer strikes a nail, that player may exclaim "Spark!" and all players, Lords & Lads, must take a drink.
```

Comment:
**any** play may call "Spark!"

Reply:
I almost made the same comment. However a reread of the full clause revealed "that player" is referring to the preceding "observed by _any player_". Though technically correct, the fact the @seanKenkeremath also got caught by it warrants a rewording. I propose "that player" simply be replaced by "they", which should be a clearer reference to the true subject of the sentence.

Reply:
Ah, yeah I did read that wrong. I read it as only the player striking the nail could call Spark.

What about ```If a spark is observed when the hammer strikes a nail, any player may call "Spark!" and all players, Lords & Lads, must take a drink.```

Also a bit of a nit, but I like the term "call" instead of "exclaim" since we are using that elsewhere like "call Uprising", "call Rally", etc